### PR TITLE
implement version increments for RX scans

### PR DESCRIPTION
**Pull request by guinneak198**

### CODE REVIEW

File: examples/run_Hahn_echo.py [hunk 1]
```diff
@@ -92,7 +92,9 @@
 )
 data.setaxis("ph1", ph1_cyc / 4)
 data.reorder(["ph1", "nScans", "t2"])
-data.set_prop("postproc_type", "spincore_SE_v1")
+data.set_prop("postproc_type", "spincore_generalproc_v1")
```
Comment (jmfranck) [inline examples/run_Hahn_echo.py]: You can't say that the post processing type is this. This is the name of a post-processing function that is a general function that is used by many different post-processing types

File: examples/run_field_dep_justMw.py [hunk 2]
```diff
@@ -65,7 +64,7 @@
 min_dBm_step = 0.5
 for x in range(len(powers)):
     dB_settings = (
-        round(10 * (np.log10(powers[x]) + 3.0) / min_dBm_step) * min_dBm_step
+        round(10 * (log10(powers[x]) + 3.0) / min_dBm_step) * min_dBm_step
```
Comment (jmfranck) [inline examples/run_field_dep_justMw.py]: Why do the import statement differently and then change this? This is just an unnecessary change.

File: examples/run_field_dep_justMw.py [hunk 3]
```diff
@@ -93,7 +92,7 @@
             break
     if p.get_power_setting() < dB_settings:
         raise ValueError("After 10 tries, this power has still not settled")
-    meter_powers = np.zeros_like(dB_settings)
+    meter_powers = zeros_like(dB_settings)
```
Comment (jmfranck) [inline examples/run_field_dep_justMw.py]: This is part of the same unnecessary change referenced above

File: examples/run_field_dep_justMw.py [hunk 4]
```diff
@@ -153,7 +152,7 @@
 sweep_data.set_units("t2", "s")
 
 sweep_data.name(config_dict["type"] + "_" + str(config_dict["field_counter"]))
-sweep_data.set_prop("postproc_type", "field_sweep_v3")
+sweep_data.set_prop("postproc_type", "spincore_generalproc_v1")
```
Comment (jmfranck) [inline examples/run_field_dep_justMw.py]: Again, you need to choose a different post-processing type.

Comment (jmfranck) [reply examples/run_field_dep_justMw.py]: It seems like it should just be Field sweep version 4

File: examples/run_largecyc_CPMG.py [hunk 5]
```diff
@@ -191,7 +191,8 @@
 data.setaxis("nEcho", r_[0 : config_dict["nEchoes"]]).setaxis(
     "ph1", ph1 / 4
 ).setaxis("ph2", ph2 / 4).setaxis("ph_overall", ph_overall / 4)
-data.set_prop("postproc_type", "spincore_generalproc_v1")
+data.set_prop("postproc_type", "spincore_diffph_SE_v2")
```
Comment (jmfranck) [inline examples/run_largecyc_CPMG.py]: Can you actually replace SE with cpmg here? I did something like this at some point, but we have had clarifying conversations about the double use of this post-processing type property.

File: examples/run_largecyc_generic_echo.py [hunk 6]
```diff
@@ -136,7 +136,8 @@
     [len(ph1), len(ph_overall), -1],
 )
 data.setaxis("ph1", ph1 / 4).setaxis("ph_overall", ph_overall / 4)
-data.set_prop("postproc_type", "spincore_generalproc_v1")
+data.set_units("t2", "s")
+data.set_prop("postproc_type", "spincore_diffph_SE_v1")
```
Comment (jmfranck) [inline examples/run_largecyc_generic_echo.py]: Shouldn't you increment the version number here because of the n scans thing?

File: examples/run_var_tau.py [hunk 7]
```diff
@@ -76,8 +77,8 @@
     ret_data=None,
 )
 mytau_axis = var_tau_data.getaxis("indirect")
-mytau_axis[0]["tau_adjust"] = tau_adjust
-mytaus_axis[0]["tau"] = tau
+mytau_axis[0]["tau_adjust"] = tau_adjust_range[0]
+mytau_axis[0]["tau"] = tau[0]
```
Comment (jmfranck) [inline examples/run_var_tau.py:81]: What is going on here? This is a non-rhetorical question to be replied to in the pr

**Review by jmfranck — CHANGES_REQUESTED** [review]

**Comment by jmfranck** [comment]
I requested several changes but once those are taken care of we should be good to merge

File: examples/run_var_tau.py (hunk 7 above)
Comment (guinneak198) [reply examples/run_var_tau.py:81]: if you compare the red line 79 to 80 you see there is a typo, these changes just address. Also tau_adjust_range and tau are lists and so we want to place the first index of the list into the first index of the indirect - not the whole list

File: examples/run_field_dep_justMw.py (hunk 4 above)
Comment (guinneak198) [reply examples/run_field_dep_justMw.py]: I compared to the load_data in the master branch and there is not a field_sweep_v3 yet so this change is actually not necessary!

File: examples/run_largecyc_generic_echo.py (hunk 6 above)
Comment (guinneak198) [reply examples/run_largecyc_generic_echo.py]: again checked with load_data on master - v2 and v3 already exist and are not appropriate for this data
